packages/pwa-kit-runtime/src/utils/ssr-proxying.js

```diff
@@ -106,10 +106,12 @@ export class Headers {
                 if (key === SET_COOKIE) {
                     this.headers[key] = values.slice()
                 } else {
-                    this.headers[key] = values.split(/,\s*/).map((value) => value.trim())
+                    console.log('~109 values', values)
+                    this.headers[key] = values.split(/,\s*/).map((value) => value?.trim())
```

Comment:
the error in MRT logs was originally something about `.trim`, added optional chaining to attempt to remediate. Just a debug attempt